### PR TITLE
MetalGreymons, Meicrackmon & Cherubimon Evil.

### PR DESCRIPTION
**Pull request by AshleyPikachu**
The following fixes have been applied to
MetalGreymonVIrus -> MetalGreymonVirus
Added Name code to MetalGreymon Vaccine called "MetalGreymon"
Added name code to regular Meicrackmon.
Added name code to Cherubimon Evil called "Cherubimon"

### CODE REVIEW

**Comment by Lycanium-Z** [comment]
Ill review

File: mods/digimon/random-teams.js [hunk 1]
```diff
@@ -850,6 +850,7 @@ class RandomDigimonTeams extends RandomTeams {
 				signatureMove: "Lightning Javelin",
 			},
 			"Meicrackmon": {
+				name: "Meicrackmon",
```
Comment (Lycanium-Z) [inline mods/digimon/random-teams.js]: Unnecessary

File: mods/digimon/random-teams.js [hunk 2]
```diff
@@ -865,13 +866,14 @@ class RandomDigimonTeams extends RandomTeams {
 				signatureMove: "Berserk Thinking",
 			},
 			"MetalGreymonVaccine": {
+				name: "MetalGreymon",
 				species: "metalgreymonvaccine",
```
Comment (Lycanium-Z) [inline mods/digimon/random-teams.js]: Is there a metalgreymonvaccine? Im not sure...

Comment (AshleyPikachu) [reply mods/digimon/random-teams.js]: yes there are two vaccine and virus, its just so the sprite displays correctly as its labeled metalgreymonvaccine on zarels end so any regular metalgreymon that is not the virus one is metalgreymon vaccine, the other files should be tweaked to account for this.

**Comment by Volco** [comment]
just saying the part of a mon's object 
``"MetalGreymonVaccine": {
	name: "MetalGreymon",
	species: "metalgreymonvaccine",
...
}`` 
the very top of that the ``"MetalGreymonVaccine": {`` should be ``metalgreymonvaccine: {`` this goes for every mon btw they should all be lowercase, non-strings at the object name declaration

**Comment by Volco** [comment]
@AshleyPikachu bump

**Comment by Volco** [comment]
bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump bump

**Comment by LycaniumZ** [comment]
Posting for Ashley, everythings been tested and works @volco

File: mods/digimon/random-teams.js (hunk 2 above)
Comment (LycaniumZ) [reply mods/digimon/random-teams.js]: This is fine

**Pull request merged**